### ISSUE TITLE
publish cherry_picker on PyPI ?

### ISSUE THREAD

**Issue by matrixise** [python/core-workflow#114]
we have a `setup.py` file, maybe we could use a `core dev` account on PyPI and publish it.

**Comment by brettcannon** [comment]
@dstufft @ncoghlan there isn't some python-dev account that already exists is there?

**Comment by dstufft** [comment]
Not that I'm aware of, but I've never been involved with python-dev publishing anything to PyPI.

**Comment by Mariatta** [comment]
Maybe I'll start by uploading to https://testpypi.python.org/ first 😃  and see how that goes

**Comment by matrixise** [comment]
maybe we could start to publish to PyPI ;-) we are contributors/core dev, maybe we could publish officially our tools. I think it's a shame to clone a repository if we want to use a tool.

just `pip install core-workflow` and we could use the tools.

**Comment by brettcannon** [comment]
So a question for @dstufft for the pip developers account on PyPI: do you have issues with people trying to reset the password since the email address seems to be a public group and thus I would assume anyone could take over the account?

**Comment by brettcannon** [comment]
@matrixise we probably won't do such a broad project as some tools are specific to core devs and others apply to anyone submitting a PR, core dev or not

**Comment by dstufft** [comment]
@brettcannon It probably makes the most sense to use personal accounts unless you're planning on automating the uploads, then use a ``python-dev`` account. In either case, I wouldn't put the password to a public list.

Long term we should have real organization support for PyPI and it could just be owned by an org instead.

**Comment by Mariatta** [comment]
I uploaded cherry_picker to testpypi https://testpypi.python.org/pypi/cherry_picker/ 

:thinking: Do I need to reformat the readme differently?

**Comment by dstufft** [comment]
@Mariatta You must have a rendering error, try:

```console
$ pip install readme_renderer
$ python setup.py check -rs
``

That should give you more information.

**Comment by Mariatta** [comment]
Thanks! 😃 It's rendering better now.

**Comment by gpshead** [comment]
pypi org support would be awesome... but until that happens so long as we have 3+ personal accounts of devs listed as owners that should be sufficient.

**Comment by brettcannon** [comment]
I can be listed as one of the collaborators, @Mariatta , if you want.

**Comment by Mariatta** [comment]
> I can be listed as one of the collaborators, @Mariatta , if you want.

Ok 😄  I just don't know how to do it 😐  I've never uploaded anything to PyPI.

**Comment by willingc** [comment]
@Mariatta I've found twine is the easiest to use for uploading to PyPI. Docs are pretty good too.

**Comment by brettcannon** [comment]
https://packaging.python.org/tutorials/distributing-packages/ if you need help publishing to PyPI (and it will mirror what @willingc said about using twine). And once you register the project you can to to pypi.python.org and manage who is listed as a collaborator for the project.

**Comment by Mariatta** [comment]
With some help from @brettcannon, cherry-picker is now in PyPI 😄 
https://pypi.python.org/pypi/cherry-picker

`pip install cherry_picker`

**Comment by willingc** [comment]
@Mariatta Cool! Do you want to close this issue now?

**Comment by Mariatta** [comment]
Thanks @willingc :) I'll get this closed with #130
